clients/admin-ui/src/features/system/system-form-declaration-tab/PrivacyDeclarationFormTab.tsx

```diff
@@ -0,0 +1,273 @@
+    );
+    return handleSave(updatedDeclarations);
+  };
+
+  const handleCloseForm = () => {
+    setShowForm(false);
+    setCurrentDeclaration(undefined);
+  };
+
+  const handleCreateDeclaration = async (
+    values: PrivacyDeclarationResponse
+  ) => {
+    if (checkAlreadyExists(values)) {
+      return undefined;
+    }
+
+    toast.closeAll();
+    const updatedDeclarations = [...system.privacy_declarations, values];
+    const res = await handleSave(updatedDeclarations);
+
+    handleCloseForm();
+    return res;
+  };
+
+  const handleOpenNewForm = () => {
+    setShowForm(true);
+    setCurrentDeclaration(undefined);
+  };
+
+  const handleOpenEditForm = (
+    declarationToEdit: PrivacyDeclarationResponse
+  ) => {
+    setShowForm(true);
+    setCurrentDeclaration(declarationToEdit);
+  };
+
+  const handleSubmit = (values: PrivacyDeclarationResponse) => {
+    handleCloseForm();
+    if (currentDeclaration) {
+      handleEditDeclaration(currentDeclaration, values);
+    } else {
+      handleCreateDeclaration(values);
+    }
+  };
+
+  const handleDelete = async (
+    declarationToDelete: PrivacyDeclarationResponse
+  ) => {
+    const updatedDeclarations = system.privacy_declarations.filter(
+      (dec) => dec.id !== declarationToDelete.id
+    );
+    return handleSave(updatedDeclarations, true);
+  };
+
+  // Reset the new form when the system changes (i.e. when clicking on a new datamap node)
+  useEffect(() => {
+    setShowForm(false);
+  }, [system.fides_key]);
+
+  return (
+    <Stack spacing={6}>
+      {system.privacy_declarations.length === 0 ? (
+        <Box
+          display="flex"
+          flexDirection="row"
+          p={4}
+          mt={6}
+          border="1px"
+          borderColor="blue.400"
+          borderRadius={8}
+          backgroundColor="gray.50"
+        >
+          <HStack spacing={2} display="flex" alignItems="start">
+            <WarningTwoIcon color="blue.400" boxSize="18px" />
+            <Stack spacing={1}>
+              <Heading as="h4" size="md">
+                You don&apos;t have a data use set up for this system yet.
+              </Heading>
+              <Text size="sm">[copy]</Text>
+              <HStack>
+                <Button
+                  variant="outline"
+                  size="md"
+                  data-testid="add-btn"
+                  onClick={handleOpenNewForm}
+                  mt={2}
+                  {...addButtonProps}
+                >
+                  Add data use
+                </Button>
+              </HStack>
+            </Stack>
+          </HStack>
+        </Box>
+      ) : (
+        <PrivacyDeclarationDisplayGroup
+          heading="Data use"
+          declarations={system.privacy_declarations}
+          handleAdd={handleOpenNewForm}
+          handleEdit={handleOpenEditForm}
+          handleDelete={handleDelete}
+        />
+      )}
+      {unassignedCookies ? (
```

Comment:
```suggestion
      {unassignedCookies && system.cookies.length  > 0 ? (
```
this is showing up when there are no cookies. It looks a little weird having the empty section

right now this section shows up when there